src/main/modules/session/index.ts

```diff
@@ -27,13 +27,14 @@ export class SessionStorage {
 
   private getStore(app: Application): session.Store {
     const redisHost: string = config.get('session.redis.host');
+    const redisPort: number = config.get('session.redis.port');
     const redisPass: string = config.get('session.redis.key');
 
-    if (redisHost && redisPass) {
+    if (redisHost && redisPort && redisPass) {
```

Comment:
Why do we need to check the redisPort in the condition? Further down you are doing a check fir null too "redisPort ?? 6380"

Reply:
Fixed in https://github.com/hmcts/idam-user-dashboard/pull/108/commits/5885dfa7362f02f8982715bacc8cbe1a9b942454